test/spec/vanilla.spec.js

```diff
@@ -160,16 +172,17 @@ describe('backend with vanillajs', function() {
   it('should not serve up mock data when params do not match', function () {
```

Comment:
suggestions? Throwing an exception is not serving up the mock data. It also is a nice opposite of the previous test "should serve up mock data when params match".